bands/configspec.ini

```diff
@@ -2,6 +2,8 @@
 band_email = string(default="MAGFest Music Department <music@magfest.org>")
 band_email_signature = string(default="- MAGFest Music Department")
 
+require_dedicated_band_table_presence = boolean(default=True)
```

Comment:
Ehhh, I don't like this config option. This should be stuff that's done via model_checks, which is overridable in plugins. Take this config option out and I'll go ahead and do the refactoring necessary to make it work for Magstock.